CRM/Financial/BAO/Payment.php

```diff
@@ -188,7 +188,6 @@ public static function create(array $params): CRM_Financial_DAO_FinancialTrxn {
           'is_post_payment_create' => TRUE,
           'is_email_receipt' => $params['is_send_contribution_notification'],
           'trxn_date' => $params['trxn_date'],
-          'trxn_id' => $params['trxn_id'] ?? NULL,
```

Comment:
@mattwire so where does this get concatenated onto the existing value - `completeOrder` makes sense as a place to concatenate it on but I don't see it there

Reply:
It's concatenated in `CRM_Financial_BAO_Payment::updateRelatedContribution()`.

Reply:
OK thanks - if jenkins is happy then that covers it!